### YamlMime:PythonClass
uid: azure.iot.device.exceptions.CredentialError
name: CredentialError
fullName: azure.iot.device.exceptions.CredentialError
module: azure.iot.device.exceptions
inheritances:
- azure.iot.device.exceptions.ClientError
summary: Could not connect client using given credentials
constructor:
  syntax: CredentialError(message=None, cause=None)
